app/ui/LargeButton.tsx

```diff
@@ -21,6 +22,7 @@ export function LargeButton(props: {
         borderRadius: 12,
         fontSize: 12,
         border: "none",
+        width: width !== undefined ? width : "auto",
```

Comment:
I think you should be able to pass width straight in without using the ternary

Reply:
yep